modules/manage/pages/security/authorization.adoc

```diff
@@ -174,7 +174,7 @@ To get this information at the CLI, run:
 rpk acl --help-operations
 ----
 
-In flag form to set up a general producing/consuming client, you can invoke `rpk acl create` up to three times with the following (including your `--allow-principal`):
+In flag form to set up a general producing/consuming client, you can invoke xref:reference:rpk/rpk-acl-create.adoc[`rpk acl create`] up to three times with the following (including your `--allow-principal`):
```

Comment:
```suggestion
In flag form to set up a general producing/consuming client, you can invoke xref:reference:rpk/rpk-acl/rpk-acl-create.adoc[`rpk acl create`] up to three times with the following (including your `--allow-principal`):
```
I think this needed one more subdirectory

Reply:
good catch!